app/code/Bulbulatory/Recomendations/Model/Total/Quote/Custom.php

```diff
@@ -0,0 +1,53 @@
+<?php
+namespace Bulbulatory\Recomendations\Model\Total\Quote;
+
+use Magento\Quote\Model\Quote\Address\Total\AbstractTotal;
+use Magento\Framework\Pricing\PriceCurrencyInterface;
+use Magento\Quote\Model\Quote;
+use Magento\Quote\Api\Data\ShippingAssignmentInterface;
+use Magento\Quote\Model\Quote\Address\Total;
+use Bulbulatory\Recomendations\Block\Recommendation\Index;
+
+class Custom extends AbstractTotal
```

Comment:
Możemy sobie dodawać wiele takich "customowych" zniżek czy opłat, więc dobrze byłoby nazwać klasę (i plik) w taki sposób, aby było jasne za co jest ta klasa odpowiedzialna - np. `Recommendation` czy `RecommendationDiscount` od razu to wskazuje.